config/codon.config

```diff
@@ -55,6 +55,10 @@ process {
         cpus = 1
         memory = { 1.GB * task.attempt }
     }
+    withName: GENERATE_COMBINED_QC_REPORT {
+        cpus = 1
+        memory = { 20.MB * task.attempt }
```

Comment:
I would use at least 100MB, there is some overhead that is not accounted for with nextflow

Reply:
Done in https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/commit/bbdadc23853c5f36d54ac9e42876c3cf7b3450e4